### ISSUE TITLE
"Import class" and "Download Sources" don't work after update

### ISSUE THREAD

**Issue by metametadata** [cursive-ide/cursive#2309]
Started happening after updating `IntelliJ IDEA Community` to `2019.3.3` and `Cursive` to `1.9.1-eap4-2019.3`.

# Problems

1) I no longer see `Import class` in intention dropdowns, e.g.:

    <img width="641" alt="Screenshot 2020-02-15 12 21 40" src="https://user-images.githubusercontent.com/4689913/74586199-f1281200-4fed-11ea-98f8-83a89cf5556f.png">

2) When navigating to Java source and clicking "Download Sources":

    <img width="322" alt="Screenshot 2020-02-15 12 24 53" src="https://user-images.githubusercontent.com/4689913/74586224-2f253600-4fee-11ea-8518-a8da43040ee7.png">

    The IDE reports an error: `Write access is allowed from event dispatch thread only` and there are some `cursive.` mentions in the stacktrace:

```
Write access is allowed from event dispatch thread only

com.intellij.openapi.diagnostic.RuntimeExceptionWithAttachments: EventQueue.isDispatchThread()=false Toolkit.getEventQueue()=com.intellij.ide.IdeEventQueue@3f9f315f
Current thread: Thread[Downloading Sources...,4,Idea Thread Group] 417761045
SystemEventQueueThread: Thread[AWT-EventQueue-0,6,Idea Thread Group] 1263753130
	at com.intellij.openapi.application.impl.ApplicationImpl.assertIsDispatchThread(ApplicationImpl.java:959)
	at com.intellij.openapi.application.impl.ApplicationImpl.startWrite(ApplicationImpl.java:1041)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteActionWithClass(ApplicationImpl.java:871)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:894)
	at cursive.application.ApplicationKt.withWrite(Application.kt:11)
	at cursive.leiningen.sources.LeiningenAttachSourcesProvider$getActions$1$perform$task$1.run(LeiningenAttachSourcesProvider.kt:90)
	at cursive.build.BuildSystemUtil$runTask$1$1.run(BuildSystemUtil.kt:93)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:888)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.lambda$run$0(CoreProgressManager.java:402)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:221)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:402)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

Accompanying [threadDump.txt](https://github.com/cursive-ide/cursive/files/4208356/threadDump.txt).

# Workarounds

* Downgrading _only_ `IDEA` to `2019.3.2` (i.e. while still using `Cursive 1.9.1-eap4-2019.3`) fixes problem 1) but not 2).

* Downgrading _only_ `Cursive` to `1.9.0-2019.3` fixes problem 2) but not 1).

* Downgrading both IDEA and Cursive fixes both problems.

**Comment by kelveden** [comment]
(1) is a problem with IntelliJ Ultimate too. I'm also on the latest non-eap Cursive build.

I do seem to recall seeing something similar to (2) myself but I've tried again today and it seems to be fine. However, (1) is still an issue. IntelliJ metadata that might be of interest:

```
IntelliJ IDEA 2019.3.3 (Ultimate Edition)
Build #IU-193.6494.35, built on February 11, 2020
Runtime version: 11.0.6+8-b702.1 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.4.22-1-MANJARO
```
Cursive: `1.9.0-2019.3`

**Comment by metametadata** [comment]
Both issues seem to be fixed in IntelliJ IDEA 2019.3.4 (Community Edition) and Cursive 1.9.1-2019.3.
